docs/openstack-cloud-controller-manager/expose-applications-using-loadbalancer-type-service.md

```diff
@@ -180,8 +180,6 @@ Request Body:
 
   Defines whether to create health monitor for the load balancer pool, if not specified, use `create-monitor` config. The health monitor can be created or deleted dynamically. A health monitor is required for services with `externalTrafficPolicy: Local`.
 
-  Not supported when `lb-provider=ovn` is configured in openstack-cloud-controller-manager.
```

Comment:
Can we change this to say that ETP=Local is supported with OVN starting from Wallaby? https://opendev.org/openstack/ovn-octavia-provider/commit/8bbd8f1b62d7811433079bff8ce940fbdd041a0c